InfinispanAPIObject.adoc

```diff
@@ -0,0 +1,36 @@
+== Infinispan API
+
+=== Motivation
+
+Being able to offer a simple API to create clustered caches and improve API usability
+
+
+== Infinispan
+
+This factory will help to create a cluster of Infinispan.
+
+Whenever a new member is added on the VM, the implicit configuration will handle the visibility between members.
+
+```java
+
+public final class Infinispan {
+
+    private Infinispan() {}
+
+    //Constructs and starts a new instance of the CacheManager, using the system defaults.
+    public static EmbeddedCacheManager createClusteredCache() {}
```

Comment:
This is the style of interface I'd like to see.
Additionally I'd like to see a way to piggy-back a  new instance of "Infinispan" on an existing one, so things like transport, thread-pools, etc can be shared:
```java
static Infinispan newInstance(Infinispan other);
```